Source/Core/Common/GL/GLInterface/AGL.mm

```diff
@@ -5,9 +5,8 @@
 #include "Common/GL/GLInterface/AGL.h"
 #include "Common/Logging/Log.h"
 
-static bool UpdateCachedDimensions(NSView* view, u32* width, u32* height)
-{
-  NSWindow* window = [view window];
+static bool UpdateCachedDimensions(NSView *view, u32 *width, u32 *height) {
```

Comment:
I know it is a style thing so not everyone will agree, but why is it putting the curly on the same line?  I am not a huge fan of that approach.  Was this not enforced before?

It seems odd that some of these would do curly on the next line and some would do curly on same line.. we should be consistent.  Are we missing something in our config?